ATTRIBUTION.md

```diff
@@ -17,6 +17,75 @@ This package contains a variety of materials, including data sets and related ma
 * Location: ***prereise/gather/data/ba_to_county.txt***
 
 
+#### US County Map Shape Files
+#### Source
+* Name: US County map
+* Author: US Census Bureau
+* Description: US county map
+* Source: https://www.census.gov/
+* Exact source location: https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html
+#### Destination
+* Modifications to source files(s): None
+* Location: ***prereise/gather/data/cb_2020_us_county_500k.zip***
+
+
+#### Retail Service Territory 
+#### Source
+* Name: Electric Retail Service Territory 
+* Author: Homeland Infrastructure Foundation-Level Data (HIFLD)
+* Description: electric retail service territory
+* Source: https://hifld-geoplatform.opendata.arcgis.com/
+* Exact source location: https://hifld-geoplatform.opendata.arcgis.com/datasets/geoplatform::electric-retail-service-territories-2/explore
+#### Destination
+* Modifications to source files(s): none
+* Location: ***prereise/gather/data/Electric_Retail_Service_Territories.zip***
+
+
+#### Upstate Subregions of NYISO
+#### Source
+* Name: upstate subregions of NYISO
+* Author: ArcGIS
+* Description: upstate subregions of NYISO
+* Source: https://www.arcgis.com/
+* Exact source location: https://www.arcgis.com/home/item.html?id=6fd1de467b134f47a607721f23a69f0c
+#### Destination
+* Modifications to source files(s): None for the upstate subregions A-F, they are directly downloaded by the program 
+* Location: ***prereise/gather/data/remap_ba_area/nyiso/***
+
+#### Downstate Subregions of NYISO
+#### Source
+* Name: downstate subregions of NYISO
+* Author: US Census Bureau
+* Description: downstate subregions of NYISO
+* Source: https://www.census.gov/
+* Exact source location: https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html
+#### Destination
+* Modifications to source files(s): Downstate subregions G_I are made by grouping and splitting county shapes.
+* Location: ***prereise/gather/data/remap_ba_area/nyiso/NYISO_downstate.geojson***
+
+
+#### CAISO
+#### Source
+* Name: subregions of CAISO
+* Author: CAISO
+* Description: the subregions of CAISO
+* Source: https://www.caiso.com/
+* Exact source location: https://cecgis-caenergy.opendata.arcgis.com/datasets/CAEnergy::california-electric-balancing-authority-areas
+#### Destination
+* Modifications to source files(s): select county shapes from the image map
+* Location: ***prereise/gather/data/remap_ba_area/CA_balancing_authorities/***
```

Comment:
Are we selecting counties from the map here, or using the shapefiles exactly as we download them?

Reply:
We use the downloaded file as is for CA.

Reply:
If so, then we should update the modifications text which current says `select county shapes from the image map`.

Reply:
Will change to none.